core/jobs.py

```diff
@@ -33,8 +33,8 @@
 from mapreduce import context
 from mapreduce import input_readers
 from mapreduce import mapreduce_pipeline
-from mapreduce import output_writers
 from mapreduce import model as mapreduce_model
+from mapreduce import output_writers
```

Comment:
@anookeen: Please do not change the order of imports. (They are already sorted)  
This is the reason why the lint checks are failing.

Reply:
Ok I will update it. @apb7

Reply:
@apb7 when I keep sorting order as given in oppia develop branch it gives sorting error and doesn't pass linting tests locally. 
![image](https://user-images.githubusercontent.com/20276300/36046286-5007ce58-0dd0-11e8-9373-5c31241ff5d5.png)
![image](https://user-images.githubusercontent.com/20276300/36046298-5b14dd7c-0dd0-11e8-9f1c-475dee358865.png)

Reply:
@anookeen: I think you are going wrong somewhere. This is not possible since all files on the develop branch (with no change) pass the linter tests.  
Please see _carefully_ if you're changing the order of imports.

Reply:
![image](https://user-images.githubusercontent.com/20276300/36046555-3c453486-0dd1-11e8-9bc9-e816f82f1cdc.png)
This is my order of imports @apb7

Reply:
@anookeen: This order should not cause an import error. Can you create a [gist](https://gist.github.com/) of the file and share it here. Also, can you please check the spacing after each group of imports and see if it matches with that of the develop branch?

Reply:
ok @apb7

Reply:
https://gist.github.com/anookeen/6d1b3bc3d2fcdd26c739cfabe975bd7d 
here is gist

Reply:
@anookeen: Please end your file with a new line. This seems to be the cause of the problem.

Reply:
updated gist @apb7 still the same import sorting errors are shown

Reply:
@anookeen: On line 1814 (the last line), there is still an extra space. Remove that too and try again.

Reply:
Remving the space gives 
![image](https://user-images.githubusercontent.com/20276300/36050524-b63d555e-0dde-11e8-89ec-d1b1469b7b88.png)

updated gist @apb7

Reply:
@anookeen: Please remove one single space and not the complete line. (In the present gist, go to the last line 1813 and just press the enter key. Make it similar to the end of the original file). Re-run the tests.